A3.md

```diff
@@ -45,40 +45,42 @@ One downside to this approach is the potential for the tracer to bloat memory. F
 The implementation will be done in the C-core first, with Java and Go to follow. The tracer object will hold a linked list of trace node and a linked list of subchannel tracers. Each of the subchannel tracers will hold their own linked lists of trace nodes.
 
 ```
+// One node of tracing data
 struct grpc_trace_node {
   grpc_slice* data;
   grpc_error* error;
   gpr_timespec time;
   grpc_connectivity_state connectivity_state;
   grpc_trace_node* next;
+
+  // this last field is used for efficient garbage collection of the
+  // subchannels. Once the node that traces a subchannel going inactive is
+  // overwritten, the tracing object for that subchannel is freed.
+  grpc_subchannel_tracer* free_this_subchannel;
```

Comment:
I would just call this "subchannel".

Also, there is no more grpc_subchannel_tracer type, so this should probably be a grpc_channel_tracer.